### PR TITLE
Speaker lineup

### PR DESCRIPTION
**Pull request by brandenbyers**
Here is the initial speaker lineup. I'd like to put out an announcement tomorrow morning.

### CODE REVIEW

File: events/smkmeetup/index.html [hunk 1]
```diff
@@ -32,6 +32,37 @@ <h2>Get ready for the 5th semi-annual <a href="https://twitter.com/search?f=twee
...
+        <h4>Tiny</a></h4>
+        <p>Tinymakesthings started making keycaps in 2016. She specializes in sculpting custom clay caps but has recently started to venture into the resin casting space. She loves to share her passion for mechanical keyboards and often livestreams her creative process on Twitch.</p>
+      </div>
+    </section>
+    <section class="spotlight">
+      <div class="image"><a href="https://talljoe.com/about/"><img src="/images/speaker-joe-wasson.jpg" alt="Joe Wasson" /></div><div class="content">
+          <h4>Joe Wasson</a></h4>
+        <p>I am a software engineer and team lead. I started my career at Microsoft in the mid 90’s and have worked for a variety of companies large and small, most recently as the Director of Product for payments at Xero, Inc. As part of finding “a better way”, I have built and programmed a more than 2-dozen mechanical keyboards in various layouts and have contributed to the QMK Firmware project. I’m currently learning stenography in attempt to further decrease friction in the human/machine interface.</p>
```
Comment (jetpacktuxedo) [inline events/smkmeetup/index.html]: This feels weird, jumping from third to first person

**Comment by brandenbyers** [comment]
It is common to have a speaker lineup without talk descriptions. Many speakers don’t complete their talks until the last week before a conference, so trying to get final talk description and title is better towards the end. Plus, it avoids things like the artisan-turned-design talk where he changed his topic last minute and we got feedback from some people that they came for the talks specifically to hear him talk about artisans (which we had listed) and then he didn’t.

File: events/smkmeetup/index.html (hunk 1 above)
Comment (brandenbyers) [reply events/smkmeetup/index.html]: I agree. That’s how the speakers sent their bios. You think we should re-write the first person ones?

Comment (jetpacktuxedo) [reply events/smkmeetup/index.html]: I think we should tweak either the third person or first person ones so they match. Not particularly concerned with which ones.

**Comment by jetpacktuxedo** [comment]
I guess I don't go to enough conferences, but it seems like it would be hard to decide what talks to go to (if any) if you have no idea what they are about...

**Comment by brandenbyers** [comment]
We will have a talk schedule posted a week or two before the event. That solves that issue, right?

**Pull request merged**